README.md

```diff
@@ -9,32 +9,13 @@ information out of PDF files.
 
 ## Tasks
 
-### `Extract Text Data With RPA`
+### `Extract Text Data`
 
-Extract textual data with the local help of `RPA.PDF` library.
-
-Watch demo: https://www.loom.com/share/4bc28045f45941d18d935a60e35e227a
+Extract textual data from PDF file.
 
 > Usually this is sufficient for most of the cases.
 
 
-### `Extract Tabular Data With Camelot`
-
-Extract tables with the Camelot library. (see external
-[dependency](https://pypi.org/project/camelot-py/))
-
-Watch demo: https://www.loom.com/share/87d434dad8d748ada566a8c199cbea90
-
-> This is useful for getting out nicely formatted tabular data, but comes at the cost
-> of heavier dependencies brought in the built environment.
-
-### `Extract Structured Data With AI`
-
-Extract fields detected in both text or image-based PDFs using 3rd-party external
-services wrapped by the `RPA.DocumentAI` library.
-
-Watch demo: https://www.loom.com/share/e9753b884b6f4aa1ac0271580a0cf682
+### `Extract element from table in PDF`
```

Comment:
The "`" makes sense here if we decide to hold in the header the name of the task itself, like `Extract Elements`, otherwise we can drop them as it doesn't refer to an existing task defined in the _robot.yaml_.